### PR TITLE
Call QuadtreePrimitive.tileLoadProgressEvent after rendering frame state

### LINKED ISSUE #3450: Use frameState.afterRender to fire QuadtreePrimitive.tileLoadProgressEvent
In case, for example, someone adds new primitives in the callback.

For example, see https://github.com/AnalyticalGraphicsInc/cesium/blob/7d449906cf65cf06e75d5cde7bf2dd5921bc95dd/Source/Scene/ModelAnimationCollection.js#L387

### PR DESCRIPTION
**Pull request by bampakoa**
@pjcozzi This issue was inactive for some time and I decided to take on that, if it is ok.

### CODE REVIEW

**Comment by hpinkos** [comment]
Thanks @bampakoa!  Pull requests are always welcome
@bagnell, do you want to review this?

File: Source/Scene/QuadtreePrimitive.js [hunk 1]
```diff
@@ -496,7 +496,13 @@ define([
...
         }
 
-        raiseTileLoadProgressEvent(primitive);
+        frameState.afterRender.push(createTileProgressFunction(primitive));
+    }
+
+    function createTileProgressFunction(primitive) {
+        return function() {
+            raiseTileLoadProgressEvent(primitive);
+        }
```
Comment (hpinkos) [inline Source/Scene/QuadtreePrimitive.js]: You have a `jsHint` error because you're missing a `;` here.
You can run jsHint locally with `npm run jsHint`

**Comment by hpinkos** [comment]
We'll probably want a unit test for this too.  @bampakoa could you write one?

**Comment by bampakoa** [comment]
Absolutely @hpinkos . I will push it to the current PR ASAP.

**Comment by bampakoa** [comment]
This is ready

**Comment by pjcozzi** [comment]
Thanks @bampakoa, looks good at quick glance!

@rahwang could you please review and test?

**Comment by pjcozzi** [comment]
@bampakoa also, please update [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md).

**Comment by bampakoa** [comment]
@pjcozzi I will close this PR, because of an erroneous commit, and I will submit a new one,.

**Pull request closed without merging**